python/opae.admin/opae/admin/fpga.py

```diff
@@ -380,17 +380,16 @@ class fpga_base(class_node):
     PORT_PATTERN = 'intel-fpga-port.*'
     PCI_DRIVER = 'intel-fpga-pci'
     SYSFS_CLASS = 'fpga'
-    BOOT_TYPES = ['bmcimg', 'fpga']
+    BOOT_TYPES = ['bmcimg', 'retimer']
     BOOT_PAGES = {
-        (0x8086, 0x0b30): {'fpga': {'user': 1,
-                                    'factory': 0},
-                           'bmcimg': {'user': 0,
-                                      'factory': 1}},
-        (0x8086, 0x0b2b): {'fpga': {'user': 0,
-                                    'factory': 1},
-                           'bmcimg': {'user': 1,
-                                      'factory': 0}},
-        (0x8086, 0x09c4): {'fpga': {'user': 0}}
+        (0x8086, 0x0b30): {'bmcimg': {'user': 0,
```

Comment:
BOOT_PAGES  can be removed, valid for Production driver

Reply:
There is still logic that accesses these structures.  I have instrumented the table so that the code flows through unscathed.